esp/esdllib/esdl_def.cpp

```diff
@@ -1278,8 +1278,8 @@ class EsdlDefFile : public CInterface, implements IEsdlDefFile
 class EsdlDefServiceArrayIterator : public CInterface, implements IEsdlDefServiceIterator
 {
 private:
-      ESDLServiceArray & array;
-      aindex_t             cur;
+    ESDLServiceArray & array;
+    aindex_t             cur;
```

Comment:
If you insert multiple spaces between the type and name, as if to align a series of types and names, then the names should be aligned.